lib/DDG/Spice/Spotify.pm

```diff
@@ -0,0 +1,22 @@
+package DDG::Spice::Spotify;
+
+# ABSTRACT: Music search on Spotify
+
+use strict;
+use DDG::Spice;
+
+spice is_cached => 0;
+spice proxy_cache_valid => "200 1d";
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://api.spotify.com/v1/search?q=$1&market=$2&limit=35&type=track';
```

Comment:
@JMPerez we might want to specify that $1 is the artist? `madonna` surface the track "Madonna" by Drake which it probably shouldn't